Improved file extraction workflows and continuous integration. Enhanced documentation with new properties (#738)
KnowledgeCaptureAndDiscovery/somef: README.md

- **Acknowledgement**: Text acknowledging funding sources or contributors
- **Application domain**: The application domain of the repository. Current supported domains include: Astrophysics, Audio, Computer vision, Graphs, Natural language processing, Reinforcement learning, Semantc web, Sequential. Domains are not mutually exclusive. These domains have been extracted from [awesome lists](https://github.com/topics/awesome-list) and [Papers with code](https://paperswithcode.com/). Find more information in our [documentation](https://somef.readthedocs.io/en/latest/)

> - **Author**: Person or Organization responsable of the project.

Reviewer: ```suggestion
- **Author**: Person or Organization responsible of the project. We also detect the authors of papers associated with the repository (through the reference publication)
```

- **Contribution guidelines**: Text indicating how to contribute to this code repository
- **Contributors**: Contributors to a software component
- **Creation date**: Date when the repository was created

> - **DateCreated**: Date of creation.

Reviewer: ```suggestion
```
This is literally the one above (creation date)

> - **DateModified**: Date of last release.

Reviewer: ```suggestion
- **Date modified**: Date of last release.
```

> - **DatePublished**: Date of first publication.

Reviewer: ```suggestion
- **Date published**: Date when the project was first published (first release).
```
